player_comparator.py

```diff
@@ -87,12 +69,16 @@ def _build_players_list_from_unique_ids(self, play_by_play_dataframe, unique_ids
 
     def _extract_player_name(self, player_id, player_column_number, pbp):
 
-        id = f'PLAYER{player_column_number}_ID'
-        name = f'PLAYER{player_column_number}_NAME'
+        id = f'player{player_column_number}_id'
```

Comment:
If this string format is something that could change, you might want to have functions that take in a player object and return the `id` or the `name` in this format. 

That way those functions can be tested and ensured to be consistent.